Runtime/BacktraceClient.cs

```diff
@@ -1318,5 +1314,13 @@ private bool ShouldSkipReport(ReportFilterType type, Exception exception, string
                 || (SkipReport != null && SkipReport.Invoke(type, exception, message));
 
         }
+
+        internal IList<string> GetNativeAttachments()
+        {
+            return _clientReportAttachments.ToList()
```

Comment:
is the first `.ToList()` here necessary?

Reply:
you're right - we don't need it. Thank you.